transport-native-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -114,7 +141,11 @@ public static FileDescriptor newBlockingEventFd() {
         return new FileDescriptor(blockingEventFd());
     }
 
-    public static native void ioUringExit(RingBuffer ringBuffer);
+    public static native void ioUringExit(long submissionQueueArrayAddress, int submissionQueueRingEntries,
+                                          long submissionQueueRingAddress, int submissionQueueRingSize,
+                                          int submissionQueueRingFd,
+                                          long completionQueueRingAddress, int completionQueueRingSize,
+                                          int completionQueueRingfd);
```

Comment:
I don't think we need to pass the ringFd twice here or submissionQueueRingSize, and we can pass completionQueueRingEntries instead of completionQueueRingSize.

i.e. number of args can be reduced by 2

Reply:
I think it would better to pass the `ringFd` that to the RingBuffer

Reply:
forget it, as sqe and cqe require ring_fd :)

Reply:
I would prefer to keep it mostly as it is for clarity.